### PR TITLE
Containerize germline-dna

### PR DESCRIPTION
**Pull request by rhpvorderman**

### CODE REVIEW

File: dockerTags.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+biopet-scatterregions: "0.2--0"
+biopet-vcfstats: "1.2--0"
+# A mulled container is needed to have both picard and bwa in one container.
+# quay.io/biocontainers/mulled-v2-002f51ea92721407ef440b921fb5940f424be842
+# This container contains: picard (2.18.7), bwa (0.7.17-r1188)
+"bwa+picard": "43ec6124f9f4f875515f9548733b8b4e5fed9aa6-0"
+cutadapt: "1.16--py36_2"
+fastqc: "0.11.7--4"
+gatk4: "4.1.0.0--0"
+#htseq: "0.9.1--py36h7eb728f_2"
```
Comment (DavyCats) [inline dockerTags.yml]: htseq isn't involved in DNA analysis at all, why is this here?

File: sample.wdl [hunk 2]
```diff
@@ -36,8 +37,9 @@ workflow Sample {
             bamFiles = library.bqsrBamFile,
             gvcfPath = sampleDir + "/" + sample.id + ".g.vcf.gz",
             dbsnpVCF = dbSNP,
-            regions = regions
-    }
+            regions = regions,
+            dockerTags = dockerTags
+        }
```
Comment (DavyCats) [inline sample.wdl]:
```suggestion
    }
```

File: tests/test_germline_dna.yml [hunk 3]
```diff
@@ -49,5 +49,5 @@
     - path: "test-output/multisample_variants/multisample.vcf.gz.tbi"
     - path: "test-output/multisample_variants/stats"
     - path: "test-output/samples/NA12878/lib_U0a/NA12878-U0a.markdup.bam"
-    - path: "test-output/samples/NA12878/lib_U0a/NA12878-U0a.markdup.bai"
```
Comment (DavyCats) [inline tests/test_germline_dna.yml]: Why don't we check for the index anymore?

File: tests/test_germline_dna.yml [hunk 4]
```diff
@@ -49,5 +49,5 @@
...
-    - path: "test-output/samples/NA12878/lib_U0a/metrics/NA12878-U0a.markdup.flagstats"
\ No newline at end of file
+    - path: "test-output/samples/NA12878/lib_U0a/metrics/NA12878-U0a.markdup.flagstats"
+    - path: "test-output/samples/NA12878/lib_U0a/metrics/wgs2-paired-end-lib1.markdup.flagstats"
```
Comment (DavyCats) [inline tests/test_germline_dna.yml]: This doesn't seem to be correct, the sample and library names earlier in this path don't match the ones in the file name. Actually, it looks like this line shouldn't be here at all.

File: dockerTags.yml (hunk 1 above)
Comment (rhpvorderman) [reply dockerTags.yml]: Remnant from RNA-seq.Fixed

File: tests/test_germline_dna.yml (hunk 3 above)
Comment (rhpvorderman) [reply tests/test_germline_dna.yml]: I worked on an outdated branch by accident. The merging caused some issues. Fixed this

File: tests/test_germline_dna.yml (hunk 4 above)
Comment (rhpvorderman) [reply tests/test_germline_dna.yml]: Merge error.  Removed it.

**Comment by rhpvorderman** [comment]
@DavyCats 
.gitignore is not necessary anymore. All its ignores were related to the scala/sbt framework. So I deleted it. pytest only generates `__pycache__` files these cab be in the gitignore at some point.

**Comment by DavyCats** [comment]
@rhpvorderman Didn't the `.gitignore` contain `.idea`?

**Comment by rhpvorderman** [comment]
@DavyCats 
Editor specific files should be in global gitignore. Best to make the project editor agnostic.
Otherwise the gitignore will grow with every editor that is used on the project.
